src/clients/meta/MetaClient.h

```diff
@@ -24,6 +24,14 @@
 
 DECLARE_int32(meta_client_retry_times);
 
+namespace nebula {
+namespace storage {
+class ExtMetaClientModifier;
+class MetaClientTestUpdater;
+class ChainAddEdgesTest_AddEdgesLocalTest_Test;
+}  // namespace storage
+}  // namespace nebula
```

Comment:
It seems these forward declarations are not necessary and I don't understand why to make the MetaClient to be the friend of ChainAddEdges?